### ISSUE TITLE
Several Contrast errors present in the 'Details' section (report) of each search result

### ISSUE THREAD

**Issue by csmith-he** [archesproject/Arches_HER#1030]
**Steps to Reproduce:**
1. Navigate to Search page 
2. Enter a valid search term
3. Select the 'Details' section of any search result
4. Enable WAVE browser extension
5. View contrast errors

**Expected Result:**
No contrast errors should be present in the page.

**Actual Result:**
Contrast errors are present in the Details section (the number of errors varies depending on search result).
